.buildkite/pipeline.yml

```diff
@@ -0,0 +1,28 @@
+steps:
+  - label: ":go: Run check-static"
+    command: "make check-static"
+    agents:
+      image: "golang:1.19.5"
+      cpu: "8"
+      memory: "4G"
+
+  - label: ":go: Run unit tests"
+    command: "make test-go-ci"
+    artifact_paths:
+      - "build/test-results/*.xml"
+      - "build/test-coverage/*.xml"
```

Comment:
Not able to find a way to show this coverage data or how to generate the report for Buildkite as it was done in Jenkins.